src/utils.js

```diff
@@ -66,15 +66,15 @@ function createCSVDownload(columns, data, options) {
     ? options.onDownload(buildHead, buildBody, columns, data)
     : `${CSVHead}${CSVBody}`.trim();
 
-  if (options.onDownload && csv === false) {
```

Comment:
I think we still need this, as this allows people to switch off download functionality by returning `false` in the callback.

Reply:
Fixed in 9343413843cd3d375aa6f3ad0799f2f0135342e9.